### PR TITLE
Feat: Change robots.txt - reco SEO Agency Primelis

### PR DESCRIPTION
**Pull request by eddysegaud**
[WWW-992]

[WWW-992]: https://bonitasoft.atlassian.net/browse/WWW-992

### CODE REVIEW

File: antora-playbook.yml [hunk 1]
```diff
@@ -3,7 +3,10 @@ site:
   # features that depend on the site URL: https://docs.antora.org/antora/2.3/playbook/site-url/#when-should-the-site-url-be-set
   url: https://documentation.bonitasoft.com/
   start_page: bonita::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /0/archives/release-notes*
```
Comment (tbouffard) [inline antora-playbook.yml:8]: ⚠️ this page doesn't exist and so generates a 404 error, so I don't understand why it is allowed 😸 
https://documentation.bonitasoft.com/0/archives/release-notes

Note that the page does not exist in the bonita component either: https://documentation.bonitasoft.com/bonita/0/archives/release-notes.html

If we have links leading to such urls, they probably come from a bad redirects configuration and we should fix them.

File: antora-playbook.yml [hunk 2]
```diff
@@ -3,7 +3,10 @@ site:
...
+    Disallow: */0/archives*
```
Comment (tbouffard) [inline antora-playbook.yml:9]: ❓ I guess they are disallowed as we have configured redirects from old bonita 5.x versions to the archive pages. And we won't to drop references to the old versions in the Search Engine.
It is the reason of the change?

**Comment by tbouffard** [comment]
⚠️ https://bonitasoft-bonita-documentation-site-build_preview-pr-246.surge.sh/robots.txt content with commit is 1850874
```
User-agent: *
Disallow: /
```
This is the normal behavior: the script used to generated the preview always set the robots configuration to `disallow` in the Antora playbook.
See https://github.com/bonitasoft/bonita-documentation-site/blob/1850874b2d892c27f278ed8cec54e831f3e20d89/scripts/generate-content-for-preview-antora-playbook.js#L126-L127

File: antora-playbook.yml (hunk 2 above)
Comment (eddysegaud) [reply antora-playbook.yml:9]: Hello, the reason is detail in this  Primelis document : https://docs.google.com/presentation/d/1xr9hc-sYbVZ3PTrjP_OjV9JDEo-NGuEx5eXgVY_NRxI/edit
Pages 21 to 26 and 29

We will must change urls path for archives pages.
The change in the robots.txt is the first step.
Actual change on the file has no effect on the presence in search engine.

**Comment by eddysegaud** [comment]
> ℹ️ https://bonitasoft-bonita-documentation-site-build_preview-pr-246.surge.sh/robots.txt content with commit is [1850874](https://github.com/bonitasoft/bonita-documentation-site/commit/1850874b2d892c27f278ed8cec54e831f3e20d89)
> 
> ```
> User-agent: *
> Disallow: /
> ```
> 
> This is the normal behavior: the script used to generated the preview always set the robots configuration to `disallow` in the Antora playbook.
> See
> 
> https://github.com/bonitasoft/bonita-documentation-site/blob/1850874b2d892c27f278ed8cec54e831f3e20d89/scripts/generate-content-for-preview-antora-playbook.js#L126-L127
> 
> 
> Commenting this line in the preview script generates the expected robots.txt

Ok, thank you for this insight.

**Comment by tbouffard** [comment]
Closing as this does not fit the current site content.
Will be rework later.

**Pull request closed without merging**